tests/test_contracts.py

```
@@ -0,0 +1,497 @@
+        "MANAGER_PRIVATE_KEY": "0xPrivate",
+        "STRATEGIST_PRIVATE_KEY": "0xPrivate",
+        "CURATOR_PRIVATE_KEY": "0xPrivate",
+        "ORION_VAULT_ADDRESS": "0xVault",
+    }
+    with patch.dict(os.environ, env_vars):
+        yield
+
+
+def test_load_contract_abi_success():
+    """Test successful ABI loading."""
+    pass
```

Ruff (0.14.10)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)